mitodl/odl-video-service
# Dorska/subtitles deletion modal

#### What are the relevant tickets?
Fixes #284 .

#### What's this PR do?
Adds a modal dialog for deleting subtitles.

#### How should this be manually tested?
1. Ensure you have a video that has subtitles.
2. Go to the video detail page for the video and click the trash button next to a subtitles file.
3. Ensure that a confirmation dialog appears, and that it looks reasonable.
4. Ensure that canceling the dialog preserves the subtitles file.
5. Ensure that accepting the dialog deletes the subtitles file.
6. Ensure that the dialog functions normally after other dialogs are opened and closed.
